ext/dom/tests/bug81433.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Bug #81433 (DOMElement::setIdAttribute(attr, true) called twice removes ID)
+--SKIPIF--
+<?php
+require_once('skipif.inc');
+if (LIBXML_VERSION >= 20912) die('skip For libxml2 < 2.9.12 only');
```

Comment:
Why is this test limited to libxml2 < 2.9.12?

Reply:
I'm not sure of that. I copy pasted it from another test.

Reply:
Okay, in that case you can safely drop it here.